src/tools/mesh_tester.cpp

```diff
@@ -79,15 +110,36 @@ struct Input
   void fixOutfilePath();
 };
 
+const std::set<rajaPolicy> Input::s_validPolicies({
```

Comment:
@bmhan12 -- This interface seems a bit awkward to me.

What do you think about changing this to ``runtime policy``, with a default of ``sequential`` (non-raja) ?

The user can then set the policy to one of the following:
* seq           // default -- always available
* raja-seq    // if AXOM_USE_RAJA
* raja-omp  // if AXOM_USE_RAJA and AXOM_USE_OPENMP
* raja-cuda  // if AXOM_USE_RAJA and AXOM_USE_CUDA

Reply:
I think that is a good idea!

Reply:
About to push this change -- running some tests locally to make sure I got everything right.